src/pages/jaewonyang/Main/Comments/CommentsJaeWon.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import CommentJaeWon from '../Comment/CommentJaeWon';
+import './CommentsJaeWon.scss';
+
+function Comments() {
+  const [commentInput, setCommentInput] = useState('');
+  const [commentList, setCommentList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setCommentList(data);
+      });
+  }, []);
+
+  const handleCommentInput = event => {
+    setCommentInput(event.target.value);
+  };
+
+  const onClick = event => {
+    event.preventDefault();
+    if (commentInput.trim() !== '') {
+      let newArr = [...commentList];
```

Comment:
참조형 데이터이기 때문에 const로 선언해도 내부 데이터 수정 가능
```suggestion
      const newArr = [...commentList];
```